Fish-Registration/Startup.cs

```diff
@@ -27,11 +27,11 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 
-            // services.AddDbContext<ApplicationDbContext>(options =>
-            //     options.UseNpgsql(Configuration.GetConnectionString("DefaultConnection")));
-
             services.AddDbContext<ApplicationDbContext>(options =>
-                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
+                options.UseNpgsql(Configuration.GetConnectionString("DefaultConnection")));
+
+            // services.AddDbContext<ApplicationDbContext>(options =>
```

Comment:
@drkrieger1 you'll want to uncomment this line and comment the line that uses Npgsql.

Reply:
...when it is merged to master.